src/tests/double_array_structure.rs

```diff
@@ -98,3 +98,76 @@ fn test_input_order() {
     assert_eq!(pma_sorted.states, pma_unsorted.states);
     assert_eq!(pma_sorted.outputs, pma_unsorted.outputs);
 }
+
+#[test]
+fn test_n_blocks_1_1() {
+    let mut patterns = vec![];
+    // state 0: reserved for the root state
+    // state 1: reserved for the dead state
+    // base = 0xff; fills 0x02..=0x7f
```

Comment:
```suggestion
    // base = 0xfe; fills 0x02..=0xff
```
`base` will become `0xfe` because `0xfe` and `0xff` are acceptable and the smaller will be used.